Tests for text_strip fail on \xa0

`pytest` output:

```
[gw3] darwin -- Python 3.8.2 /Users/uwe/miniconda3/envs/fletcher/bin/python

fletcher_variant = 'chunked'

>   ???

tests/test_text.py:320:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests/test_text.py:352: in test_text_strip
    _do_test_text_strip(fletcher_variant, 1, data)
tests/test_text.py:376: in _do_test_text_strip
    tm.assert_series_equal(result_fr, result_pd)
pandas/_libs/testing.pyx:64: in pandas._libs.testing.assert_almost_equal
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   AssertionError: Series are different
E
E   Series values are different (100.0 %)
E   [index]: [0]
E   [left]:  [ ]
E   [right]: []

pandas/_libs/testing.pyx:178: AssertionError
------------------------------------------------------------------------------------------------ Hypothesis ------------------------------------------------------------------------------------------------
Falsifying example: test_text_strip(
    fletcher_variant='chunked', data=['\xa0'],
```

--- a/fletcher/algorithms/string_builder.py
+++ b/fletcher/algorithms/string_builder.py
@@ -42,7 +42,7 @@ malloc_callable = LibcMalloc()  # type: Any
 malloc = malloc_callable if os.getenv("NUMBA_DISABLE_JIT", "0") != "1" else malloc_nojit
 
 
-@numba.jitclass(
+@numba.experimental.jitclass(
     [
         ("ptr", numba.types.voidptr),
         ("capacity", numba.int64),
@@ -191,7 +191,7 @@ class ByteVector:
         self.buf = numba.carray(self.ptr, self.capacity, numba_byte)
 
 
-@numba.jitclass(
+@numba.experimental.jitclass(
     [
         ("ptr", numba.types.voidptr),
         ("capacity", numba.int64),
@@ -267,7 +267,7 @@ def byte_for_bits(num_bits):
     return (num_bits + 7) // 8
 
 
-@numba.jitclass(
+@numba.experimental.jitclass(
     [
         ("length", numba.uint64),
         ("null_count", numba.uint64),

--- a/fletcher/string_array.py
+++ b/fletcher/string_array.py
@@ -457,7 +457,7 @@ class TextAccessor(TextAccessorBase):
         # for whatever reason 0x200B (zero width space) is not considered a space by pandas.split()
         if to_strip is None:
             to_strip = (
-                " \t\r\n\x1f\x1e\x1d\x1c\x0c\x0b"
+                " \t\r\n\x85\x1f\x1e\x1d\x1c\x0c\x0b\xa0"
                 "\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2000\u2009\u200A\u2028\u2029\u202F\u205F"
             )
         return self._series_like(_text_strip(self.data, to_strip))

--- a/tests/test_text.py
+++ b/tests/test_text.py
@@ -328,6 +328,8 @@ def test_text_strip_offset(fletcher_variant, fletcher_slice_offset, data):
         [" a"],
         ["a "],
         [" a "],
+        # https://github.com/xhochy/fletcher/issues/174
+        ["\xa0"],
         ["\u2000a\u2000"],
         ["\u2000\u200C\u2000"],
         ["\n\u200C\r"],

